osf/models/user.py

```diff
@@ -389,6 +389,9 @@ class OSFUser(DirtyFieldsMixin, GuidMixin, BaseModel, AbstractBaseUser, Permissi
 
     chronos_user_id = models.TextField(null=True, blank=True, db_index=True)
 
+    # This is taken via logging into CAS
+    department = models.TextField(null=True, blank=True)
```

Comment:
Probably a comment on what this field is:

> The primary department to which the institution user belongs, in case we support multiple departments in the future.

Or, we can rename it as `primary_department` w/o extra comment.